rust/processor/src/processors/fungible_asset_processor.rs

```diff
@@ -367,6 +374,21 @@ impl ProcessorTrait for FungibleAssetProcessor {
         let (coin_balance, fa_balance): (Vec<_>, Vec<_>) = current_unified_fungible_asset_balances
             .into_iter()
             .partition(|x| x.is_primary.is_none());
+
+        if self
+            .deprecated_tables
+            .contains(TableFlags::FUNGIBLE_ASSET_BALANCES)
+        {
+            fungible_asset_balances.clear();
+        }
+
+        if self
+            .deprecated_tables
+            .contains(TableFlags::CURRENT_FUNGIBLE_ASSET_BALANCES)
+        {
+            current_fungible_asset_balances.clear();
```

Comment:
Just a note: this table is on a different deprecation timeline than the other ones for the coin -> FA migration. It's fine to add the flag, but we don't want to turn it off just yet.

Reply:
we need to re-eval this table `current_fungible_asset_balances` with @ying-w

Reply:
we don't need this in parquet, we can delete this table. according to Ying, we can replace with fungible_asset_balances

Reply:
Will start a separate migration plan for this table with the different timeline.

Reply:
Wait `fungible_asset_balances` is going to Parquet, but we need to keep `current_fungible_asset_balances` for now for the coin -> FA migration, which I'll deal with .